pkg/config/storage.go

```diff
@@ -1,72 +1,84 @@
 package config
 
 import (
+	"fmt"
+
+	"github.com/openshift/odo/pkg/localConfigProvider"
 	"github.com/pkg/errors"
 )
 
-// StorageCreate sets the storage related information in the local configuration
-func (lci *LocalConfigInfo) StorageCreate(name, size, path string) (ComponentStorageSettings, error) {
-	storage := ComponentStorageSettings{
-		Name: name,
-		Size: size,
-		Path: path,
+// GetStorage gets the storage with the given name
+func (lci *LocalConfigInfo) GetStorage(storageName string) *localConfigProvider.LocalStorage {
```

Comment:
Since there's no modification of the data, this would be better off as a function instead of being a method attached to the `LocalConfigInfo`, IMO.

Reply:
We need it to call `ListStorage()`